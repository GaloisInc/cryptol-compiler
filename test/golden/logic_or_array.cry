main = [0xFF, 0x00] || [0xD1, 0xD0]