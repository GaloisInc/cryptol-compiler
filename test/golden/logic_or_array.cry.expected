[0xff, 0xd0]
